### ISSUE TITLE
Allow to ignore rule for an entire stage with `# hadolint stage ignore`

### ISSUE THREAD

**Issue by felipecrs** [koalaman/shellcheck#3237]
It would be great if I could ignore certain rules for an entire stage with something like `# hadolint stage ignore` (similar to `# hadolint global ignore`). Example:

```dockerfile
# hadolint stage ignore=DL3004
FROM somebaseimage AS builder

RUN sudo touch /etc/someconfig.txt

FROM alpine

COPY --from=builder /etc/someconfig.txt /etc/someconfig.txt
```

It would be safer than using `# hadolint global ignore`.

**Comment by felipecrs** [comment]
Wow. My bad. I meant to open this issue in Hadolint repository instead.
